src/components/TestModal.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import Modal from 'react-bootstrap/Modal';
+import Button from 'react-bootstrap/Button';
+
+import { Link } from 'react-router-dom';
+import arrowLeft from '../image/arrow-left.svg';
+import '../CSS/Icon.css';
+
+const TestModal = ({ show, handleClose, item }) => {
+  let lastPurchase = item ? new Date(item.last_purchased).toDateString() : '';
+
+  return (
+    <>
+      <Modal show={show} onHide={handleClose}>
+        <Modal.Header>
```

Comment:
Nice use of `Modal`'s compound components 👍

Reply:
🔥